conduct/code-of-conduct.md

```diff
@@ -2,7 +2,9 @@ This is a template code of conduct that event organizers can use for conferences
 
 # Code of Conduct
 
-`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a conference intended for professional networking and collaboration in the R community. We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. 
+`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a conference intended for professional networking and collaboration in the R community. 
+
+We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. An inclusive space free of harassment encourages interaction among diverse groups. We want to make certain <EVENT> is welcoming, and encourages participants to be involved moving forward.
```

Comment:
```suggestion
We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. An inclusive space free of harassment encourages interaction among diverse groups. We want to make certain `<EVENT>` is welcoming, and encourages participants to be involved moving forward.
```

recommend putting `<EVENT>` in backticks for consistency with the rest of the doc

Reply:
done